### PR TITLE
Features/ncne withdrawal testcases

### PR DESCRIPTION
**Pull request by ssrajan**
**New Withdrawal Task page**

Remove Duration
Make chart number mandatory
Remove New Chart/New Edition header

**Withdrawal Workflow page**

Remove Duration
Remove Commit to Print
H Forms to be renamed to match landing page column header (Forms/Withdrawal Actions)
Chart number is mandatory
Chart number is read only after V1 is Completed
Change Consider Email SDR to Consider Email to SDR

### CODE REVIEW

File: src/NCNEPortal/Pages/Withdrawal.cshtml [hunk 1]
```diff
@@ -19,9 +19,9 @@
                 <img id="ukhoLogo" src="images/ukhologo.svg" alt="UKHO Logo" />
             </a>
         </div>
-        <div class="col-7 offset-1">
-            <h1>New Chart / New Edition</h1>
-        </div>
+        @*<div class="col-7 offset-1">
+                <h1>New Chart / New Edition</h1>
+            </div>*@
```
Comment (UKHOGreg) [inline src/NCNEPortal/Pages/Withdrawal.cshtml:24]: Comment?

File: src/NCNEPortal.TestAutomation.Framework/Pages/NewTaskPage.cs [hunk 2]
```diff
@@ -1,12 +1,12 @@
...
+﻿using Common.TestAutomation.Framework.Pages;
 using NCNEPortal.TestAutomation.Framework.Configs;
 using OpenQA.Selenium;
 using OpenQA.Selenium.Support.UI;
+using System;
 
 namespace NCNEPortal.TestAutomation.Framework.Pages
 {
-    public class NewTaskPage : PageBase
+    public class NewTaskPage : PageBase, ILandingPage
```
Comment (UKHOGreg) [inline src/NCNEPortal.TestAutomation.Framework/Pages/NewTaskPage.cs:9]: Forgive me, but why does `NewTaskPage` implement `ILandingPage`?

File: src/NCNEPortal.TestAutomation.Framework/Pages/WithdrawalPage.cs [hunk 3]
```diff
@@ -0,0 +1,35 @@
+﻿using Common.TestAutomation.Framework.Pages;
+using NCNEPortal.TestAutomation.Framework.Configs;
+using OpenQA.Selenium;
+using OpenQA.Selenium.Support.UI;
+using System;
+
+namespace NCNEPortal.TestAutomation.Framework.Pages
+{
+    public class WithdrawalPage : PageBase, ILandingPage
```
Comment (UKHOGreg) [inline src/NCNEPortal.TestAutomation.Framework/Pages/WithdrawalPage.cs:9]: Forgive me, but why does `WithdrawalPage` implement `ILandingPage`?

File: src/NCNEPortal.UnitTests/PageValidationTests.cs [hunk 4]
```diff
@@ -99,6 +99,28 @@ public void Validation_for_ValidateNewTaskPage_missing_WorkflowType_fails(string
 
         }
 
+        [TestCase(null)]
+        [TestCase("")]
+        public void Validation_for_ValidateNewTaskPage_missing_ChartNoe_fails_for_Withdrawal(string chartNo)
```
Comment (UKHOGreg) [inline src/NCNEPortal.UnitTests/PageValidationTests.cs:104]:
```suggestion
        public void Validation_for_ValidateNewTaskPage_missing_ChartNo_fails_for_Withdrawal(string chartNo)
```

**Review by UKHOGreg — CHANGES_REQUESTED** [review]

File: src/NCNEPortal/Pages/Withdrawal.cshtml (hunk 1 above)
Comment (ssrajan) [reply src/NCNEPortal/Pages/Withdrawal.cshtml]: Removed the commented code

File: src/NCNEPortal.UnitTests/PageValidationTests.cs (hunk 4 above)
Comment (ssrajan) [reply src/NCNEPortal.UnitTests/PageValidationTests.cs]: corrected the spelling mistake

File: src/NCNEPortal.TestAutomation.Framework/Pages/NewTaskPage.cs (hunk 2 above)
Comment (ssrajan) [reply src/NCNEPortal.TestAutomation.Framework/Pages/NewTaskPage.cs]: Its a common interface to have Navigate method. Now renamed  to IPage

File: src/NCNEPortal.TestAutomation.Framework/Pages/WithdrawalPage.cs (hunk 3 above)
Comment (ssrajan) [reply src/NCNEPortal.TestAutomation.Framework/Pages/WithdrawalPage.cs]: Its a common interface to have Navigate method. Now renamed  to IPage

File: src/Common.TestAutomation.Framework/Pages/ILandingPage.cs [hunk 5]
```diff
@@ -1,6 +1,6 @@
 ﻿namespace Common.TestAutomation.Framework.Pages
 {
-    public interface ILandingPage
+    public interface IPage
```
Comment (UKHOGreg) [inline src/Common.TestAutomation.Framework/Pages/ILandingPage.cs:3]: Filename is now incorrect ☹

**Review by UKHOGreg — CHANGES_REQUESTED** [review]

File: src/Common.TestAutomation.Framework/Pages/ILandingPage.cs (hunk 5 above)
Comment (ssrajan) [reply src/Common.TestAutomation.Framework/Pages/ILandingPage.cs:3]: Yes.  changed now

**Review by UKHOGreg — APPROVED** [review]

**Review by bonpoole — APPROVED** [review]

**Pull request merged**